app/code/community/Bolt/Boltpay/Model/BoltOrder.php

```diff
@@ -175,15 +184,12 @@ function ($item) use ($quote, &$calculatedTotal, $boltHelper) {
         } else {
 
             $billingAddress  = $quote->getBillingAddress();
+            $this->correctBillingAddress($quote);
```

Comment:
Shouldn't we correct the billing address before getting it from `quote`?

Reply:
yes.  This was a mistake that I made after changing the parameter from the billing address object to the quote object.  I forgot to change the order.